src/Parser/Mage/Shared/Modules/Features/RuneOfPower.js

```diff
@@ -0,0 +1,79 @@
+import React from 'react';
+import SPELLS from 'common/SPELLS';
+import SpellLink from 'common/SpellLink';
+import SpellIcon from 'common/SpellIcon';
+import { formatPercentage, formatNumber } from 'common/format';
+import StatisticBox, { STATISTIC_ORDER } from 'Main/StatisticBox';
+import Combatants from 'Parser/Core/Modules/Combatants';
+import AbilityTracker from 'Parser/Core/Modules/AbilityTracker';
+import Module from 'Parser/Core/Module';
+
+import getDamageBonus from '../GetDamageBonus';
+
+const DAMAGE_BONUS = 0.4;
+const RUNE_DURATION = 10;
+const INCANTERS_FLOW_EXPECTED_BOOST = 0.12;
+
+// FIXME due to interactions with Ignite, the damage boost number will be underrated for Fire Mages. Still fine for Arcane and Frost.
+class RuneOfPower extends Module {
+
+  static dependencies = {
+    combatants: Combatants,
+    abilityTracker: AbilityTracker,
+	}
+
+  damage = 0;
+
+  on_initialized() {
+	   this.active = this.combatants.selected.hasTalent(SPELLS.RUNE_OF_POWER_TALENT.id);
+  }
+
+  on_byPlayer_damage(event) {
+    if(this.combatants.selected.hasBuff(SPELLS.RUNE_OF_POWER_BUFF.id)) {
+      this.damage += getDamageBonus(event, DAMAGE_BONUS);
+    }
+  }
+
+  suggestions(when) {
+    const damagePercent = this.owner.getPercentageOfTotalDamageDone(this.damage);
+    const damageIncreasePercent = damagePercent/(1-damagePercent);
+
+    when(damageIncreasePercent).isLessThan(INCANTERS_FLOW_EXPECTED_BOOST)
+      .addSuggestion((suggest, actual, recommended) => {
+        return suggest(<span>Your <SpellLink id={SPELLS.RUNE_OF_POWER_TALENT.id}/> damage boost is below the expected passive gain from <SpellLink id={SPELLS.INCANTERS_FLOW_TALENT.id}/>. Either find ways to make better use of the talent, or switch to <SpellLink id={SPELLS.INCANTERS_FLOW_TALENT.id}/>.</span>)
+          .icon(SPELLS.RUNE_OF_POWER_TALENT.icon)
+          .actual(`${formatPercentage(damageIncreasePercent)}% damage increase from Rune of Power`)
+          .recommended(`${formatPercentage(recommended)}% is the passive gain from Incanter's Flow`)
+          .regular(recommended).major(recommended-0.03);
+      });
+
+      const casts = this.abilityTracker.getAbility(SPELLS.RUNE_OF_POWER_TALENT.id).casts;
```

Comment:
If the Rune of Power is never cast, this would return `undefined` I believe (the object returned by `getAbility()` exists, but it doesn't have the `casts` property - https://github.com/WoWAnalyzer/WoWAnalyzer/blob/5b61270b21b0bd4c0dec0f05232c5238a023db72/src/Parser/Core/Modules/AbilityTracker.js#L23 , that one is set in `cast` event handler).